Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -3664,6 +3664,18 @@ bool CStaticFunctionDefinitions::GetVehicleModelExhaustFumesPosition(unsigned sh
     return false;
 }
 
+bool CStaticFunctionDefinitions::GetVehicleWheelScale(CClientVehicle& Vehicle, float& fWheelScale)
+{
+    fWheelScale = Vehicle.GetWheelScale();
+    return true;
+}
+
+bool CStaticFunctionDefinitions::SetVehicleWheelScale(CClientVehicle& Vehicle, float fWheelScale)
+{
+    Vehicle.SetWheelScale(fWheelScale);
+    return true;
+}
```

Comment:
It looks like these can be removed :) (as well as in the header)

Reply:
Sure 👍 

I'll remove them tomorrow, although I may supersede this PR with a better one soon™️.